ClangPowerTools/ClangPowerTools/Script/ScriptConstants.cs

```diff
@@ -6,6 +6,8 @@ public class ScriptConstants
   {
     #region Constants
 
+    public const string kCppProjectGuid = "{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}";
```

Comment:
Take into consideration the above comment. This is the `GUID` representation for a `C++ Project` in VS.